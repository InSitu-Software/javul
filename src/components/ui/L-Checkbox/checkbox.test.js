import { mount } from 'vue-test-utils'
import Checkbox from './checkbox'

describe('Checkbox', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Checkbox)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
