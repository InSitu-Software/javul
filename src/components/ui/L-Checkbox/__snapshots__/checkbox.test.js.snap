// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should match the snapshot 1`] = `
<div>
  <label>
    noo...
  </label>
   
  <input
    type="checkbox"
  />
</div>
`;
